railties/lib/rails/application/bootstrap.rb

```diff
@@ -62,7 +62,9 @@ module Bootstrap
           Rails.logger = broadcast_logger
         end
 
-        unless config.consider_all_requests_local
+        if config.consider_all_requests_local
+          Rails.error.debug_mode = true
```

Comment:
Should this code be in the `initialize_logger` initializer? It isn't related to logger. I'd create a new one

Reply:
Yup. Corrected.